src/components/FeatureStyler/FeatureStyler.js

```diff
@@ -782,8 +844,16 @@ class FeatureStyler extends PureComponent {
   }
 
   renderColors(color, classNameColors, classNameSelected, onClick, label) {
-    const { t, colors, lineColors, labels } = this.props;
-    const colorsList = label === labels.modifyLineColor ? lineColors : colors;
+    const { t, colors, textBgColors, lineColors, labels } = this.props;
+    // const colorsList = label === labels.modifyLineColor ? lineColors : colors;
+    let colorsList;
+    if (label === labels.modifyTextBgColor) {
+      colorsList = textBgColors;
+    } else if (label === labels.modifyLineColor) {
+      colorsList = lineColors;
+    } else {
+      colorsList = colors;
+    }
```

Comment:
instead of this `if` pass the colors as a new  parameter of the function.
Put this parameter at the 2nd place:
```
renderColors(color, colors, classNameColors, classNameSelected, onClick, label)
```

Reply:
Done!